Source/WebCore/loader/cache/AllowedFonts.cpp

```diff
@@ -851,16 +852,20 @@ bool isFontBinaryAllowed(const void* data, size_t size, DownloadableBinaryFontAl
         return false;
     case DownloadableBinaryFontAllowedTypes::Restricted: {
         auto sha = hashForFontData(data, size);
-        return allowedFontHashesInLockdownMode().contains(sha);
+        bool allowedFontHashes;
+        allowedFontHashes = allowedFontHashesInLockdownMode().contains(sha);
```

Comment:
```suggestion
        auto allowedFontHashes = allowedFontHashesInLockdownMode().contains(sha);
```

Reply:
thanks.

Reply:
Doesn't look like this comment has been addressed yet, so marking as unresolved.

Reply:
done on my local computer. :) but fair enough.

Reply:
ok this time in the PR 😄